ui/lib/core/addon/templates/components/readonly-form-field.hbs

```diff
@@ -1,5 +1,5 @@
 <label for={{@attr.name}} class="is-label" data-test-readonly-label>
-  {{this.labelString}}
+  {{or @label this.labelString}}
```

Comment:
nit: debating whether this should be in the getter for `this.labelString`. On one hand, this makes it very clear that passing @label in explicitly will override any other calculations. But, we already have a method here for calculating the label and it could easily live there 🤔

Reply:
I had it that way initially and and thought the getter had a lot going on as well - but I'm happy to move it back there! 

Another thought is I could do something similar to the above, and create a `let` for `keyType` and leave this component alone. This  also gives us the benefit of including the option's subtext... 🤔  Any opinion?